addons/base_automation/data/base_automation_data.xml

```diff
@@ -13,4 +13,39 @@
             <field name="active" eval="False" />
         </record>
     </data>
+    <data>
```

Comment:
Just realized this isn't doable in stable, we can get rid of this action and only keep the one that opens the form view of the automated action